modules/nw-bgp-examples.adoc

```diff
@@ -0,0 +1,259 @@
+----
+
+The OVN-Kubernetes controller checks if this `FRRConfiguration` CR applies to its node as indicated by the `kubernetes.io/hostname: ovn-worker` label. For these examples a user-defined network named `blue` exists with a network of `10.0.0.0/16` and a matching VRF device exists in the Linux host. The slice of this supernet allocated to nodes with the `kubernetes.io/hostname: ovn-worker` label is `10.0.1.0/24`.
+
+[id="advertising-pod-ips-from-a-user-defined-network-over-bgp_{context}"]
+== Advertising pod IPs from a user-defined network over BGP
+
+In this scenario, the blue user-defined network is exposed to the external network so that the attached pods have IP addresses that are reachable from the external network. The user-defined networks are each attached to a different VRF device:
+
+Red user-defined network::
+- A VRF named `mp0-red`
+- A subnet of `10.0.0.0/24`
+
+Blue user-defined network::
+- A VRF named `mp0-blue`
+- A subnet of `10.0.1.0/24`
+
+In this configuration, two separate user-defined networks are defined. The red network covers the `10.0.0.0/24` subnet and the blue network covers the `10.0.1.0/24` subnet.
+
+The following `RouteAdvertisements` CR describes the configuration for the blue tenant:
+
+.`RouteAdvertisements` CR for the blue tenant
+[source,yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: RouteAdvertisements
+metadata:
+  name: default
+spec:
+  networkSelector:
+    matchLabels:
+      k8s.ovn.org/metadata.name: blue
+  frrConfigurationSelector:
+    matchLabels:
+      routeAdvertisements: default
+----
+
+When the OVN-Kubernetes controller sees this `RouteAdvertisements` CR, it generates a `FRRConfiguration` object that configures the FRR daemon to advertise the routes.
+
+.`FRRConfiguration` CR generated by OVN-Kubernetes
+[source,yaml]
+----
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  name: route-advertisements-blue
+  namespace: openshift-frr-k8s
+spec:
+  bgp:
+    routers:
+    - asn: 64512
+      vrf: blue
+      prefixes:
+      - 10.0.1.0/24
+    - asn: 64512
+      neighbors:
+        - address: 172.18.0.5
+          asn: 64512
+          toAdvertise:
+            allowed:
+              prefixes:
+              - 10.0.1.0/24
+  raw:
+    rawConfig: |-
+      router bgp 64512
+      address-family ipv4 unicast
+        import vrf blue
+        exit-address-family
+      router bgp 64512 vrf blue
+      address-family ipv4 unicast
+        import vrf default
+        exit-address-family
+  nodeSelector:
+    matchLabels:
+      kubernetes.io/hostname: ovn-worker
+----
+
+The generated `FRRConfiguration` object configures the subnet `10.0.1.0/24`, which belongs to VRF blue, is imported into the default VRF and advertised to the `172.18.0.5` neighbor. Because the `targetVRF` uses the default value, the routes are leaked and advertised in the default VRF. Additionally, routes are imported from the default VRF into the blue VRF.
```

Comment:
correct but `... configures the subnet `10.0.1.0/24`, which belongs to VRF blue, is imported into the default VRF ... ` doesn't read well to me.